quizzes/midterm.md

```diff
@@ -162,27 +202,40 @@
 	    "water"
 	  end
 	end
+	
+	class Monkey < Animal
+		puts eat
+		puts drink
+	end
 	```
 
 1. What is the difference between using `extend` and `include`?
+	extended: it can extend a instance to anouther method this can overwrite 
+
+	include: it can include a instance this does not over write the info
 
 1. Convert this into a case statement
 
 	```ruby
-	if fruit == 'apple'
+	case fruite
+	when fruit == 'apple'
 	  #eat
-	elsif fruit == 'pear'
+	when fruit == 'pear'
 	  #eat
-	elsif fruit == 'durian'
+	when fruit == 'durian'
 	  #no thanks
 	else
 	  #eat
+	end
 	```
 	
-1.  T  /  F Arrays can only hold `String` and `Hash` values?
+1.  T  /  F Arrays can only hold `String` and `Hash` values? false.. .they can hold integers as well
 
 1.  What is the scope resolution operator? 
```

Comment:
0/10 Nice try! =) 

`::` is the scope resolution operator. You may have noticed used in classes namespaced with modules.